app/components/elements/forms/toggle_component.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Elements
+  module Forms
+    # Component for a toggle-like radio button group field
+    class ToggleComponent < FieldComponent
+      def initialize(options:, **)
+        @options = options
```

Comment:
I don't love building this off options. I'd recommend slots.

Reply:
Going to address this in a follow-up PR.